src/ReleaseHistory.md

```diff
@@ -1,5 +1,10 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+## UNRELEASED (will be 2.5.0)
```

Comment:
Can we start following this standard of noting when a minor version bump is needed if you're introducing a breaking change? Help prompt whoever may be coming along for the release.

Reply:
Yes, agreed. We could consider reversioning immediate with a preview extension, as per semver. this would facilitate advance ship/testing as well.

Reply:
Should we be following the Semantic Versioning standard? It states that breaking changes should prompt a major version increment.
[https://semver.org/](https://semver.org/)

Reply:
Adding some emphasis to Chris's recommendation above, breaking changes as major version increments was also expressed as a requirement moving forward for VS. Not sure if this is something we can unilaterally decide or if it needs to be reviewed with the TC, though.

Reply:
This is a good note, let's rev the major version before we ship.